CMSgov/cmcs-eregulations
# EREGCSC-1487 -- Resources with keyword search query parameter do not load when link pasted into new tab

Resolves [EREGCSC-1487](https://jiraent.cms.gov/browse/EREGCSC-1487)

**Description**

If you copy and paste a Resources link with a keyword search query parameter into a new tab, resources do not load and the user sees the empty state.

Example: https://regulations-pilot.cms.gov/resources/?q=clinic&sort=newest 

The above page will return zero results if opened in a new tab.  It should return 24 results.

**This pull request changes:**

- Removes computed prop `multiWordQuery` being used as a method

**Steps to manually verify this change:**

1. Visit resources page in experimental deployment here: https://nlhvn5mtui.execute-api.us-east-1.amazonaws.com/dev517/resources
2. Enter "clinic" into search input and press enter or click the magnifying glass
3. Copy URL and open in new tab
4. You should see results on the page
